src/bebb/entity.mo

```diff
@@ -134,27 +141,27 @@ module {
    * Unique formats can be stored via the other tag and labelled as such
   */
   public type EntityPreviewSupportedTypes = {
-      #Jpg;
-      #Png;
-      #Glb;
-      #Gltf;
-      #Other : Text;
+    #Jpg;
```

Comment:
for a future implementation, we can see if we want to abstract this and use categories like image, video, 3Dmodel instead of file types

Reply:
Yea agreed, we could keep nesting it in the future. Though for now might keep it as is to not make it more complicated until we see how we use it